src/main/scala/thurloe/notification/NotificationMonitor.scala

```diff
@@ -425,11 +425,11 @@ class NotificationMonitorActor(val pollInterval: FiniteDuration,
                                      Map.empty
         )
 
-      case GroupAccessRequestNotification(recipientUserId, groupName, replyTos, requesterId) =>
+      case GroupAccessRequestNotification(recipientUserId, groupName, _, requesterId) =>
         thurloe.service.Notification(
           Option(recipientUserId),
           None,
-          Option(replyTos),
+          Option(requesterId),
```

Comment:
This is the functional change: for group access request emails, the Reply-To will now be the user making the group request. Everything else in the PR is syntax changes to ensure a single reply-to value.